SwissTransferResources/Images.xcassets/Icons/camera.imageset/camera.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="25" height="24" viewBox="0 0 25 24" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
The size of the icon is incorrect :/
We will (probably) have to fix this ourselves